rollup.config.js

```diff
@@ -0,0 +1,16 @@
+export default {
+    input: 'src/index.js',
+    external: [
+        '@eik/common',
+        'node-fetch',
+        'abslog',
+        'path',
+    ],
+    output: [
+        {
+            exports: 'auto',
+            format: 'cjs',
+            file: 'dist/index.cjs',
+        },
+    ],
+};
```

Comment:
Why is rollup needed? Can this module not be used directly by node?

Reply:
At the moment, the node resolving of ESM works so a ESM can import a CJS file but not the other way round. A CJS can not import an ESM. If we want to write and provide this module as en ESM and for an application written as CJS (which most are atm) to use it we must also export a CJS version of itself along with a ESM version.

The most practical way to deal with this is to write the module as ESM and export the untouched ESM code for ESM users and then upon publish of the module transpile the module to CJS to be exported as CJS for CJS projects. Thats the reason we have Rollup in here.